File: prototypes/angular-node-javascript/backend-node-javascript/src/features/city/city-repository.js
class CityRepository {
  constructor() {
    this.items =
    [
      'Cincinnati',
      'New York',
      'Knoxville',
      'London',
      'Detroit',
      'Kapuskasing',
      'Denver',
      'Burbank',
      'San Francisco',
      'Houston',
      'Atlanta',
      'Modesto'
    ];


  }

  async getItems() {
    try {
      return await new Promise((resolve) => {
        setTimeout(() => resolve(this.items), 100);
      });
    } catch (error) {
      console.error(`Error fetching persons: ${error.message}`);
      return [];
    }
  }
}

export default CityRepository;

File: backend-java/02-backend-java-prototype/src/main/java/com/ganatan/modules/city/CityRepository.java
package com.ganatan.modules.city;

import java.util.List;
import java.util.Arrays;

public class CityRepository {

	public List<City> getItems() {
		return Arrays.asList(new City(1, "Mock Cincinnati"), new City(2, "London"), new City(3, "New York"),
				new City(4, "Knoxville"));
	}
}
